### DISCUSSION TITLE
欢迎来到「掘金翻译计划」讨论区！

### DISCUSSION THREAD

**Discussion by leviding** [xitu/gold-miner#7747] [General]
## 👋 欢迎！
  这里是「掘金翻译计划」讨论区，所有人都可以在这讨论和提问。我们希望你：
  * 提出你想知道的问题。
  * 分享有趣的想法。
  * 与社区中其他成员互动。
  * 欢迎他人，思想开放。请记住这是我们一起建立的社区 💪。

  现在，你就可以在下方做一下自我介绍，并告诉我们你对「掘金翻译计划」的看法啦 👀。

## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by evanzme** [comment]
希望 2021 年，掘金翻译计划的奖品更加丰富，哈哈哈。

**Comment by PassionPenguin** [comment]
赶紧上线处理礼品兑换。

**Comment by ljhOfGithub** [comment]
https://juejin.cn/ 无法访问，挂了tizi也不行，请问是什么原因
